README.md

```diff
@@ -67,6 +71,10 @@ It is recommended to use parameters as listed above. The various `publickey` and
 The `service_provider_repository` is a repository of service providers for which you offer IdP services. The service
 configured _must_ implement the `Surfnet\SamlBundle\Entity\ServiceProviderRepository` interface.
 
+Static remote service providers can be configured in `remote.service_providers`. Keep in mind that you need to assign 
```

Comment:
It's not really clear to me what can be configured. Maybe it's a good idea to clarify what a Static remote service provider is. And why a potential user might be interested in using it.